src/apps/experimental/routes/movies/SuggestionsView.tsx

```diff
@@ -1,160 +1,51 @@
+import React, { FC } from 'react';
+import { useGetMovieRecommendations } from 'hooks/useFetchItems';
+import globalize from 'scripts/globalize';
+import Loading from 'components/loading/LoadingComponent';
+import RecommendationContainer from '../../components/library/RecommendationContainer';
+import SuggestionsItemsContainer from '../../components/library/SuggestionsItemsContainer';
+
+import { LibraryViewProps } from 'types/library';
+import { SectionsView } from 'types/suggestionsSections';
+
+const SuggestionsView: FC<LibraryViewProps> = ({ parentId }) => {
+    const {
+        isLoading,
+        data: movieRecommendationsItems
+    } = useGetMovieRecommendations(parentId);
+
+    if (isLoading) {
+        return <Loading />;
+    }
 
     return (
-        <div ref={element}>
-            <SectionContainer
-                sectionTitle={globalize.translate('HeaderContinueWatching')}
-                enableScrollX={enableScrollX}
-                items={resumeResult.Items || []}
-                cardOptions={{
-                    preferThumb: true,
-                    shape: getThumbShape(),
-                    showYear: true
-                }}
-            />
-
-            <SectionContainer
-                sectionTitle={globalize.translate('HeaderLatestMovies')}
-                enableScrollX={enableScrollX}
-                items={latestItems}
-                cardOptions={{
-                    shape: getPortraitShape(),
-                    showYear: true
-                }}
+        <>
+            <SuggestionsItemsContainer
+                parentId={parentId}
+                sectionsView={[SectionsView.ContinueWatchingMovies, SectionsView.LatestMovies]}
             />
 
-            {!recommendations.length ? <div className='noItemsMessage centerMessage'>
-                <h1>{globalize.translate('MessageNothingHere')}</h1>
-                <p>{globalize.translate('MessageNoMovieSuggestionsAvailable')}</p>
-            </div> : recommendations.map(recommendation => {
-                return <RecommendationContainer key={recommendation.CategoryId} getPortraitShape={getPortraitShape} enableScrollX={enableScrollX} recommendation={recommendation} />;
-            })}
-        </div>
+            {!movieRecommendationsItems?.length ? (
+                <div className='noItemsMessage centerMessage'>
+                    <h1>{globalize.translate('MessageNothingHere')}</h1>
+                    <p>
+                        {globalize.translate(
+                            'MessageNoMovieSuggestionsAvailable'
+                        )}
+                    </p>
+                </div>
+            ) : (
+                movieRecommendationsItems.map((recommendation, index) => {
+                    return (
+                        <RecommendationContainer
+                            // eslint-disable-next-line react/no-array-index-key
+                            key={index} // use a unique id return value may have duplicate id
```

Comment:
Maybe this could use something like this?

```suggestion
                            key={`${recommendation.CategoryId}-${index}`} // use a unique id return value may have duplicate id
```